### PR TITLE
Add checkstyle

### LINKED ISSUE #139: Checkstyle config
1. Integrated with Gradle
2. Not very strict, just a few simple rules for formatting

@nikitin-da can you do it?

### PR DESCRIPTION
**Pull request by nikitin-da**
Closes #139 
@artem-zinnatullin  please take a look. 
What rules will be helpful or not.
Shall I remove disabled rules?

### CODE REVIEW

File: storio/config/checkstyle/checkstyle.xml [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+<!--        <property name="cacheFile" value="${checkstyle.cache.file}"/>-->
+        <!-- Checks for Javadoc comments.                     -->
+        <!-- See http://checkstyle.sf.net/config_javadoc.html -->
+        <!--module name="JavadocMethod"/-->
+        <!--module name="JavadocType"/-->
+        <!--module name="JavadocVariable"/-->
+        <!--<module name="JavadocStyle"/>-->
+        <!-- Checks for Naming Conventions.                  -->
+        <!-- See http://checkstyle.sf.net/config_naming.html -->
+        <!--<module name="ConstantName"/>-->
```
Comment (artem-zinnatullin) [inline storio/config/checkstyle/checkstyle.xml]: I think, `ConstantName` can be used

**Comment by artem-zinnatullin** [comment]
It works! Cool, just add it to `.travis.yml` `./gradlew clean build connectedAndroidTest --info` -> `./gradlew clean build checkstyle connectedAndroidTest --info`

**Comment by nikitin-da** [comment]
Updated

**Comment by artem-zinnatullin** [comment]
Thanks!

**Pull request merged**